README.md

```diff
@@ -55,8 +55,12 @@ gocover full --repository-path=${REPO ROOT PATH} --cover-profile=${PATH TO}cover
 
 - Check the coverage detail at `coverage.html`
 
+- Note: Before the coverage inspection, we will check whether a _test.go file exist within each go module. 
```

Comment:
Not module, should be package, folder or directory. Module would contain many folders which is called package.